packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -4311,7 +4311,7 @@ addEventListener('fetch', event => {});`
 			mockSubDomainRequest();
 			mockUploadWorkerRequest();
 			await runWrangler(
-				"deploy --dry-run --outdir dist --define abc:'https://www.abc.net.au/news/'"
+				`deploy --dry-run --outdir dist --define "abc:'https://www.abc.net.au/news/'"`
```

Comment:
This wasn't a correct example of how a user would specify the `define` arg with this value.

Before (incorrect, note the missing quotes by the time node.js sees the value):
```sh
▶ node -e 'console.log(process.argv)' -- --define abc:'123'  
[
  '/Users/sethi/.nvm/versions/node/v18.16.0/bin/node',
  '--define',
  'abc:123'
]
```

After (correct, note the quotes in the value as expected):
```sh
▶ node -e 'console.log(process.argv)' -- --define "abc:'123'"
[
  '/Users/sethi/.nvm/versions/node/v18.16.0/bin/node',
  '--define',
  "abc:'123'"
]
```